priam/src/main/java/com/netflix/priam/scheduler/SchedulerType.java

```diff
@@ -0,0 +1,31 @@
+package com.netflix.priam.scheduler;
+
+/**
+ * Created by aagrawal on 3/8/17.
+ */
+public enum SchedulerType {
+    TIME("TIME"),CRON("CRON");
```

Comment:
I intentionally chose TIME as we can use schedulerType with CRON implementation for flush too which currently supports hour and daily.